pandas/tests/tseries/offsets/test_offsets_properties.py

```diff
@@ -97,40 +98,43 @@ def test_on_offset_implementations(dt, offset):
     assert offset.is_on_offset(dt) == (compare == dt)
 
 
-@pytest.mark.xfail(
-    reason="res_v2 below is incorrect, needs to use the "
-    "commented-out version with tz_localize.  "
-    "But with that fix in place, hypothesis then "
-    "has errors in timezone generation."
-)
+@td.skip_if_no("dateutil", "2.7")
 @given(gen_yqm_offset, gen_date_range)
 def test_apply_index_implementations(offset, rng):
     # offset.apply_index(dti)[i] should match dti[i] + offset
-    assume(offset.n != 0)  # TODO: test for that case separately
 
-    # rng = pd.date_range(start='1/1/2000', periods=100000, freq='T')
+    # TODO: Remove this `warnings.simplefilter`
```

Comment:
what are the conditions that will allow this?

Reply:
I can't remebmer why was this necessery, Ill remove it, and see if the CI passes